apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -0,0 +1,52 @@
+import statsigReporter from '../lib/util/StatsigReporter';
+
+const isInCreateAccountButtonExperiment = statsigReporter.getIsInExperiment(
+  'create_account_button',
+  'showCreateAccountButton',
+  false
+);
+
+export function showCreateAccountButton() {
+  console.log(isInCreateAccountButtonExperiment);
+  return isInCreateAccountButtonExperiment;
+}
+
+document.addEventListener('DOMContentLoaded', function () {
+  const createAccountButton = document.querySelector('#create_account_button');
+  const createAccountButtonDesktop = document.querySelector(
+    '#create_account_button.desktop'
+  );
+  const signInButtonDesktop = document.querySelector('#signin_button.desktop');
+  const hamburgerButtons = document.querySelector('#hamburger-sign-up-buttons');
+  const signUpPage = window.location.pathname.includes('/users/sign_up');
+
+  function handleWindowResize() {
```

Comment:
This function will need to stay around in the future, right? (assuming we decide to keep the create account button) Should this eventually live somewhere else?

Reply:
I'm thinking of this more as something to file in a clean up ticket, rather than asking for any changes in this PR

Reply:
Yep! I was thinking the same thing re: tackling this during clean up.

Reply:
nit: can you add a comment or include in the name of this function that this is only called for the experiment?